angular-ngrx-scss/src/app/shared/components/issue-pull-item/issue-pull-item.component.scss

```diff
@@ -0,0 +1,65 @@
+@use '../../styles/variables.scss';
+
+.item {
+  padding: 1.125rem;
```

Comment:
This is probably out of scope of this PR, but:

- It would be nice to have some predefined margin/padding values in variables - it would help us maintain a consistent layout across the app.
- It would also be cool to add a SCSS function to map px values into rems (it's more intuitive to specify the values in px and have them converted to rems imo.) Something like `@function rem($pxValue, $sizeBase = 16) {
    $remValue: ($pxValue / $sizeBase) + rem; 
    @return $remValue;
}`

Reply:
this should be a **MUST** in terms of quality code soon but, as you said, it is definitely out of scope.

Reply:
Made a ticket for this 😁 https://github.com/thisdot/starter.dev-github-showcases/issues/511